event hubs migration guide.md

```diff
@@ -0,0 +1,207 @@
+# Migration Guide (EventHubs v2 to v5)
+
+This document is intended for users that are familiar with EventHubs V2 (`@azure/event-hubs@2.x.x`) and wish 
+to migrate their application to EventHubs V5.
+
+For a more general introduction to Event Hubs, see the [readme.md](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/eventhub/event-hubs/README.md) 
+for the @azure/event-hubs package.
+
+## General changes
+
+In the interest of simplifying the API surface we've made two distinct
+clients, rather than having a single `EventHubClient`:
+* [EventHubProducerClient](https://docs.microsoft.com/en-us/javascript/api/@azure/event-hubs/eventhubproducerclient?view=azure-node-preview)
+  for sending messages.
+* [EventHubConsumerClient](https://docs.microsoft.com/en-us/javascript/api/@azure/event-hubs/eventhubconsumerclient?view=azure-node-preview) 
+  for receiving messages.
+
+We've also merged the functionality from `EventProcessorHost` into 
+`EventHubConsumerClient`, allowing `EventHubConsumerClient` to be the single
+point of entry for receiving of any type within Event Hubs.
+
+
+### Construction
+
+| In v2                                          | Equivalent in v5                                                 | Sample |
+|------------------------------------------------|------------------------------------------------------------------|--------|
+| `EventHubClient.createFromConnectionString`    | `new EventHubProducerClient()` or `new EventHubConsumerClient()` | [receiveEvents](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/eventhub/event-hubs/samples/receiveEvents.ts),  [sendEvents](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/eventhub/event-hubs/samples/sendEvents.ts) |
+| `EventHubClient.createFromAadTokenCredentials` | `new EventHubProducerClient()` or `new EventHubConsumerClient()` | [usingAadAuth](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/eventhub/event-hubs/samples/usingAadAuth.ts)
+
+
+### Receiving events (subscribe)
```

Comment:
I wouldnt suggest referring to "subscribe" in the header. If needed, you can add a one liner before the table about the fact that we now have a new name

```suggestion
### Receiving events 
```